crates/holochain_conductor_api/src/config/conductor/paths.rs

```diff
@@ -77,6 +77,12 @@ impl TryFrom<DataRootPath> for KeystorePath {
     }
 }
 
+impl From<DataRootPath> for PathBuf {
+    fn from(value: DataRootPath) -> Self {
+        value.0
+    }
+}
```

Comment:
Without this I couldn't find an easy way to convert a `DataRootPath` to a `PathBuf`.

Reply:
You shouldn't need to, since DataRootPath is shrinkwrapped, so it implements Deref

Reply:
Actually I didn't know the deref thing. That was it.